src/components/CommentSection/CommentLoad.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './commentsection.css';
+
+const CommentLoad = props => {
+    return(
+        <div className={"CommentandUsernames"}>
```

Comment:
I do not believe that you need to put your classname in braces, quotes should be fine.